### ISSUE TITLE
Design Style Guide: Improve Color Scheme Contrast Ratios

### ISSUE THREAD

**Issue by johnfrenchxyz** [duckduckgo/zeroclickinfo-goodies#3653]
There are some accessibility issues with the current design style guide and platform itself, particularly related to insufficient contrast ratios.  But before I start listing them I wanted to make sure I was doing so in the best way possible.  I read over the guidelines for contributing but I am still unsure of where suggestions like this should go.  Never-the-less, I think it is an important thing to consider for this ecosystem as a whole.

I did open [an issue you can view here in duckduckgo-colors](https://github.com/duckduckgo/duckduckgo-colors/issues/4) discussing one of the possible changes.

While the Design Style Guide is amazing in terms of aesthetics, some of the outlined elements do not meet the recommended contrast ratios [as outlined by the WCAG 2.0 guidelines](http://webaim.org/blog/wcag-2-0-and-link-colors/).

I think that the importance of accessibility warrants a discussion, but wasn't sure if this (or another repo) was the best place to start this conversation.

**Comment by moollaza** [comment]
Left a comment in the related issue. This issue isn't relevant to this repo.
